x-pack/plugins/lists/common/constants.mock.ts

```diff
@@ -58,3 +58,5 @@ export const ITEM_TYPE = 'simple';
 export const _TAGS = [];
 export const TAGS = [];
 export const COMMENTS = [];
+export const FILTER = 'name:Nicolas Bourbaki';
```

Comment:
Curious - with the filter string do we need to use `""` for values in case they have spaces or `:`? Thinking of what I ran into with the KQL formatter stuff.

Reply:
That's just a text string for the tests, but yes when you push filters to the backend you should use double quotes.